blei-lab/edward
# Doc/reorganize

#### Summary:

This PR does three things:

1. it introduces a brand-new website to replace the current one (via a custom `pandoc` workflow).
2. it introduces automatic API documentation (via `sphinx`).
3. it documents all of `Edward` in detail.

There are a few remaining tasks before we can merge this:

- [ ] finalize getting started / delving in / tutorials
- [ ] figure out what goes into `deploy.sh`
- [ ] update `README.md` to match website and updated logo

#### How to Verify:

Run all examples and tests. They still work.

Look at the website and check for issues/typos.

#### Side Effects:

The old website is deleted.

#### Documentation:

This PR adds many lines of documentation.

#### Reviewer Suggestions:

N/A
